File: 283-move-zeroes/move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        
        int n = nums.size();

        for (int i=0;i<n;i++)
        {
            if (nums[i]==0)
            {
                nums.push_back(nums[i]);
                nums.erase(find(nums.begin(),nums.end(),nums[i]));
            }
        }
    }
};

File: 283-move-zeroes/move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        
       
        int j=0, k = nums.length-1;
        for (int i=0;i<nums.length;i++)
        {
            if (nums[i]!=0)
            {
                nums[j++] = nums[i];
            }
        }
        for (int i=k;i>=j;i--)
        {
            nums[i]=0;
        }


    }
}